Source/Core/LMCHFSSResponseFileHandler.cc

```diff
@@ -180,12 +196,35 @@ namespace locust
         }
         tfFile.close();
         LDEBUG( lmclog, "Finished reading transfer function file");
-        if(!ConvertTFtoFIR(tfArray)){
+        if(!ConvertTFtoFIR(tfArray,false)){ //bool determines if TF was generated dynamically
             return false;
         }
 	fIsFIRCreated=true;
         return true;
     }
+
+    bool TFFileHandlerCore::GenerateAnalyticTFtoFIR(double initialFreq, std::vector<std::complex<double>> tfArray)
+    {
+
+	//Replaces ReadHFSSFile() in the case where a Transfer Funciton has been generated analytically
+
+        if(fIsFIRCreated)
+        {
+            return true;
+        }
+        fTFNBins=0;
+        double tfIndex;
```

Comment:
What are tfIndex, tfRealValue, and tfImaginaryValue used for  ? If they are not uses, you could remove them.

Reply:
I believe my response earlier addresses most of the comments besides the last one. You are correct that those three objects aren't used here (they're leftovers from an earlier version of the code) and have been removed.